### PR TITLE
Plane: Fetch mixer status for assessing mixer health

### PR DESCRIPTION
**Pull request by WickedShell**
Refetch the PX4IO status and use MIXER_OK bit to determine success.

### CODE REVIEW

File: ArduPlane/px4_mixer.cpp [hunk 1]
```diff
@@ -390,6 +390,11 @@ bool Plane::setup_failsafe_mixing(void)
         goto failed;
     }
 
+    if (!(ioctl(px4io_fd, PWM_STATUS, 0) & PX4IO_P_STATUS_FLAGS_MIXER_OK)) {
+        printf("Mixer failed\n");
```
Comment (magicrub) [inline ArduPlane/px4_mixer.cpp]: All this spam?

Comment (WickedShell) [inline ArduPlane/px4_mixer.cpp]: Supposed to be a hal.console->printf :) Will fix up now. Spamming is consistent and the appropriate thing.

Comment (WickedShell) [inline ArduPlane/px4_mixer.cpp]: Fixed.

**Comment by tridge** [comment]
applied with the fixes we dicussed, thanks!

**Pull request closed without merging**